Games/types/Mafia/roles/cards/MakeGuilty.js

```diff
@@ -0,0 +1,24 @@
+const Card = require("../../Card");
+const { PRIORITY_MAKE_GUILTY } = require("../../const/Priority");
+
+module.exports = class MakeInnocent extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.meetings = {
+            "Make Guilty": {
+                states: ["Night"],
+                flags: ["voting"],
+                targets: { include: ["alive"], exclude: ["self"] },
```

Comment:
Should the targets only be non-mafia members?

Reply:
Yeah this is a divergence slightly from the EM implementation I believe, but this was deliberate, in case there were some setups where there would be a potential advantage to be gained by framing your godfather (say, living town unclears only consisted of millers)
I don't think this unbalances any old framer setups so didn't see any harm leaving this in.